### PR TITLE
Creating community page

### PR DESCRIPTION
**Pull request by jsignell**
Screen Shot 2019-03-15 at 12 03 56 PM

### CODE REVIEW

File: doc/community.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+If you think something is missing or note behaving as expected, visit the issue tracker 
+for the relevant project. All projects are under the PyViz `GitHub organization <https://github.com/pyviz>`__.
+GitHub is where bugs and enhancements are logged and discussed. 
+Search for relevant existing issues to comment on, or submit your own. Use the GitHub issue 
+tracker for:
+
+  - **Bug reports** (like “The text gets cut off in panel object.”)
+  - **Documentation issues** (like “I found this section unclear”)
+  - **Feature requests** (like “I think that colorcet should have a jet colormap.”)
```
Comment (jbednar) [inline doc/community.rst]: You're trolling me! :-)

Comment (jsignell) [reply doc/community.rst]: yes :)

**Review by jbednar — APPROVED** [review]

**Comment by philippjfr** [comment]
We could include some suggestions from Matt Rocklin's blog post, which provides an excellent outline on how to file a good issue: http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports or we could just link to it.

**Comment by jbednar** [comment]
Linking to it seems sufficient to me.

**Comment by jbednar** [comment]
Looks like we both added the link at the same time!  I've removed that an am happy to merge it now.

**Comment by jsignell** [comment]
I'll just check it for look and feel.

**Comment by jsignell** [comment]
LGTM - feel free to merge.

**Pull request merged**